src/components/Drawer/Drawer.scss

```diff
@@ -56,21 +60,21 @@ $icon-size: 26px;
         outline: 0;
         transition: background-color var(--transition-duration) ease, transform 0ms ease $open-transition-duration;
 
-        &--isOpen {
+        &--isExpanded {
             transform: rotate(180deg);
         }
 
         &:hover,
         &:focus {
-            background-color: var(--drawer-extent-hover-background-color);
+            background-color: var(--drawer-expand-hover-background-color);
```

Comment:
this style is sticky after the drawer is opened until you click somewhere else. Is that intended?

Reply:
Yes, this behavior on purpose. It shows that element has the focus, and we can use keyboard navigation. It is not a part of the requirement. For me, better to keep this style, maybe make it not so bright. What do you think?

Reply:
The color works fine for me. Only if you don't need to engage with the content of the drawer only look at it, there is no hover effect anymore after you opened it first. Maybe there is no need though, and it is fine as it is.

Reply:
I think we don't need change focus from other elements when Drawer is opened or showed automatically.